DEVELOPING.md

```diff
@@ -20,18 +20,32 @@ Dai has three different numerical units: `wad`, `ray` and `rad`
 
 - `wad`: fixed point decimal with 18 decimals (for basic quantities)
 - `ray`: fixed point decimal with 27 decimals (for precise quantites, e.g. ratios)
-- `rad`: any number multiplied by a `ray`, using integer muliplication
+- `rad`: fixed point decimal with 45 decimals (result of multiplying a `wad` and a `ray`)
 
 `wad` and `ray` units will be familiar from SCD. `rad` is a new unit and
-exists to prevent precision loss in the core CDP engine. Instead of
-using fixed point multiplication (e.g. `rmul`), integer multiplication is used.
+exists to prevent precision loss in the core CDP engine.
 
 Generally, `wad` should be used additively and `ray` should be used
-multiplicatively. It usually doesn't make sense to multiply a wad by a
-wad.
+multiplicatively. It usually doesn't make sense to multiply a `wad` by a
```

Comment:
As discussed, I'd move this to the next section.

Reply:
done